src/webui/www/private/css/style.css

```diff
@@ -328,7 +335,7 @@ a.propButton img {
 /* Mocha Customization */
 
 #mochaToolbar {
-    height: 29px;
+    height: auto !important;
```

Comment:
Well we all know `!important` should be used sparsely. So I hope we can try to cut the usage down.
Try adding `toolbarHeight: auto,` after https://github.com/qbittorrent/qBittorrent/blob/0f40fad74d6e366b16195bbf89659a0eb1cd131e/src/webui/www/private/scripts/mocha-init.js#L159 and https://github.com/qbittorrent/qBittorrent/blob/0f40fad74d6e366b16195bbf89659a0eb1cd131e/src/webui/www/private/scripts/mocha-init.js#L1129

Reply:
I see there are still a few `!important` instances but my time is limited. I'll look at them later.